tests/sample_output.txt

```diff
@@ -550,15 +550,27 @@ ConfigurationFileTestCase::test_multiple_invalid_keys:208
 ---------------------------------------------------------
 
 * Tokens with invalid values:
-  Caseimportance: Lowest (type: 'choice')
-  Status: Invalid (type: 'choice')
+  Caseimportance: Lowest
```

Comment:
This file update was done with a complete run of `tests/test_testimony.sh`, and is a complete diff of the output of all `testimony` executions done in the testing script.